packages/commonwealth/client/scripts/views/components/SublayoutHeader/useUserMenuItems.tsx

```diff
@@ -111,6 +112,12 @@ const useUserMenuItems = ({
     },
   );
 
+  const client = OpenFeature.getClient();
+  const myCommunityStakePageEnabled = client.getBooleanValue(
```

Comment:
So we should almost always be using useFlag. This will re-render the react component if the flag changes (We re-fetch flags every 2 minutes).

The only time we should be using getBooleanValue is for situations where we are evaluating a feature flag outside of a react component. The only time I have found this necessary so far is at the react-router level.

Reply:
Thanks for the help guys, i updated to use `useFlag`